tracker_depth_node_back:
  ros__parameters:
    use_sim_time: false
    debug: false
    yolo_model: yolov8m-seg.pt
    input_topic: /back_camera/image_raw
    result_topic: /back_yolo_result
    result_image_topic: /back_yolo_image
    conf_thres: 0.25
    iou_thres: 0.45
    max_det: 300
    tracker: bytetrack.yaml
    device: ""
    result_conf: true
    result_line_width: 1
    result_font_size: 1
    result_font: Arial.ttf
    result_labels: true
    result_boxes: true
    input_depth_topic: /back_camera/depth
    yolo_result_topic: /back_yolo_result_for_cloud
    camera_parameters: [479.24761962890625, 479.24761962890625, 337.1943359375, 179.7568817138672]